### PR TITLE
Add a few notes to the sub-org-admin guidelines

### PR DESCRIPTION
**Pull request by mirca**
This PR adds a few notes for suborgs:

- adds a link to this document: https://google.github.io/gsocguides/mentor/notes-for-first-year-organizations as a recommendation reading for potentially new sub-orgs. 
- mentions that: "OpenAstronomy usually assigns one slot for first year sub-orgs.
That may change in a case-by-case basis, e.g., the sub-organisation already has
experience with similar coding outreach programmes, such as [ESA-SOCIS][ESA-SOCIS]."

The remaining changes are mostly fixing minor typos, adding commas, etc.

### CODE REVIEW

File: gsoc/suborg_guidelines.md [hunk 1]
```diff
@@ -13,20 +13,22 @@ slots we will get till the student selection process ends. However,
 it's our intention to keep the process fair for the suborganisations
 and students.
 
-## I want to have to become a sub-org admin, what do I do?
+## I want to become a sub-org admin, what do I do?
```
Comment (bsipocz) [inline gsoc/suborg_guidelines.md]: is it sub-org admin though, or rather about a new sub-org?

File: gsoc/suborg_guidelines.md [hunk 2]
```diff
@@ -13,20 +13,22 @@ slots we will get till the student selection process ends. However,
...
 
-First you need to be a member of the OpenAstronomy team. That process
+First, we recommend you to carefully read [Google's notes for first year organizations][Google-notes],
```
Comment (bsipocz) [inline gsoc/suborg_guidelines.md]: point to the psf guidelines here, too. https://python-gsoc.org/mentors.html#sub-orgs, or in fact the whole mentors page is a good resource

File: gsoc/suborg_guidelines.md [hunk 3]
```diff
@@ -13,20 +13,22 @@ slots we will get till the student selection process ends. However,
...
+which provides a general idea about the goals and concepts underpinning the Google Summer of Code program.
+Then, you need to be a member of the OpenAstronomy team. That process
 is simple (and free!). You need to be an open-development organisation
 and be related with astronomy. Then make a pull-request to [our repository][OA repository]
-adding a logo, a short description, etc. 
+adding a logo, a short description, etc.
 Follow the instructions in the example at the bottom of [the `memembers.yaml` file][members PR].
```
Comment (bsipocz) [inline gsoc/suborg_guidelines.md]: I feel the OA council should come up with a better set of rules of what needed to be come a member. But don't have perfect be the enemy of good, let's go ahead with this for now as it's better already than it was.

File: gsoc/suborg_guidelines.md [hunk 4]
```diff
@@ -56,21 +58,21 @@ To add your own, you will have to create a pull-request to [our repository][OA r
...
 ### Mentors
 
-For each project idea you need to have two mentors that are committed to participate
+For each project idea you need to have at least **two** mentors that are committed to participate
 in the programme. The mentors need most importantly be familiar with the repository,
 normally they are real core-contributors, they need to be familiar with the idea and
 have some idea on how that could be implemented.
 
-The mentors also need to have time for this!!! a lot of time!! We expect around 
+The mentors also need to have time for this!!! a lot of time!! We expect around
```
Comment (bsipocz) [inline gsoc/suborg_guidelines.md]: This is a bit too informal for my taste, and a whole lot more exclamation marks than comfortable 😅  
 but ok to leave as is for now

File: gsoc/suborg_guidelines.md [hunk 5]
```diff
@@ -133,38 +136,47 @@ has, you will ask for a number of slots to the Open Astronomy administrators.
...
+will generally be favored over those that do not.
 
 The process will be as open as possible including only the admins and mentors
-involved in the selection. Remember, we cannot disclose any information on student 
+involved in the selection. Remember, we cannot disclose any information on student
 selection to the students before Google announces the selected students
 
+Note that OpenAstronomy usually assigns one slot for first year sub-orgs.
+That may change in a case-by-case basis, e.g., the sub-organisation already has
+experience with similar coding outreach programmes, such as [ESA-SOCIS][ESA-SOCIS].
```
Comment (bsipocz) [inline gsoc/suborg_guidelines.md]: or the suborg participated on their own or under another umbrella in gsoc before. in fact the socis participation is not as strong a prior in my mind (given it's much more relaxed in all sense) than a privious gsoc one.

File: gsoc/suborg_guidelines.md (hunk 1 above)
Comment (mirca) [reply gsoc/suborg_guidelines.md]: Let's go for "new sub-org"?

File: gsoc/suborg_guidelines.md (hunk 2 above)
Comment (mirca) [reply gsoc/suborg_guidelines.md]: awesome!

File: gsoc/suborg_guidelines.md (hunk 4 above)
Comment (mirca) [reply gsoc/suborg_guidelines.md]: haha, makes sense

File: gsoc/suborg_guidelines.md (hunk 5 above)
Comment (mirca) [reply gsoc/suborg_guidelines.md]: done :)

**Comment by bsipocz** [comment]
Thanks @mirca for the follow up. It all looks good to me, I leave it to @dpshelio to have a look and merge.

File: gsoc/suborg_guidelines.md (hunk 5 above)
Comment (dpshelio) [reply gsoc/suborg_guidelines.md]: well... relaxed from which point of view? as an org-admin is more stressful than GSOC. It feels more like a :roller_coaster:

**Comment by dpshelio** [comment]
Thanks @mirca!!

File: gsoc/suborg_guidelines.md (hunk 5 above)
Comment (bsipocz) [reply gsoc/suborg_guidelines.md]: OK, I see the point. For me from the outside it seems less rigorous, that is rightfully adds stress, but e.g. doesn't test that the sub-org is actually keeps the deadlines, regular meetings with students, not selecting their own students, etc. So SOCIS experience is better than nothing, but still doesn't act as proof for me to see that they can handle multiple students well as a sub-org.

**Pull request merged**